app/Http/Resources/ClientResource.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+/**
+ * Model -> OAuthClient
+ */
+class ClientResource extends JsonResource
+{
+    public function toArray($request) {
+        return [
+            'id'   => $this->id,
+            'name' => $this->name,
```

Comment:
Maybe add URL (+ ToS/Privacy URL)?

Reply:
Isn't this a little bit too much for every GET request on statuses?

Reply:
No I don't think so. A few strings won't hurt. :D
We send way more information with every event, train, etc. It's more efficient to send the url(s) for the client, rather than have another client request the urls separately.